src/Common/src/System/Security/Cryptography/DerEncoder.cs

```diff
@@ -841,7 +881,7 @@ private static bool IsPrintableStringCharacter(char c)
             return false;
         }
 
-        private static byte[] ConcatenateArrays(byte[][][] segments)
+        private static byte[] ConcatenateArrays(params byte[][][] segments)
```

Comment:
It is called from EncodeOctetString to concatenate the byte [][] returned from SegmentedEncodeOctetString. Same thing happens with EncodeUtcTime and SegmentedEncodeUtcTime